src/styles/_catalog.scss

```
@@ -1,22 +1,29 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
```

I think, it is not appropriate to repeat one element. try just 200px column

---

agree, don't use repeat for only one column
```suggestion
  grid-template-columns: 200px;
```